apps/src/storage/dataBrowser/LibraryTable.jsx

```diff
@@ -3,12 +3,51 @@ import {connect} from 'react-redux';
 import React from 'react';
 import FontAwesome from '../../templates/FontAwesome';
 import msg from '@cdo/locale';
+import color from '../../util/color';
 import {showPreview} from '../redux/data';
 import PreviewModal from './PreviewModal';
 
 const styles = {
   tableName: {
-    cursor: 'pointer'
+    fontFamily: '"Gotham 7r", sans-serif',
+    cursor: 'pointer',
+    color: color.dark_charcoal
+  },
+  preview: {
+    backgroundColor: color.background_gray,
+    borderColor: color.lighter_gray,
+    fontFamily: '"Gotham 4r", sans-serif',
+    fontSize: '14px',
```

Comment:
It looks like there is considerable overlap between some of these style object. Can we de-duplicate them?

Reply:
Yes we probably can, though I'd prefer to do that in a separate PR when this is more feature-complete- some of these styles may still change and there are some components that will likely share styles that I haven't built yet.